### DISCUSSION TITLE
cri-dockerd for ubuntu 23.10

### DISCUSSION THREAD

**Discussion by ShahidAnsari0** [Mirantis/cri-dockerd#339] [Q&A]
Can cri-dockerd be available for Ubuntu 23.10?

**Comment by neersighted** [comment]
The packaging in this repository only targets Ubuntu LTS versions. I would suggest waiting for Ubuntu Noble (24.04), due for release next month, if you need a newer Ubuntu version.

In the mean time, if you really need cri-dockerd for Ubuntu 23.10 (note: this is a distro with a very short lifetime, so I assume you know what you're doing/this isn't a production system), it is trivial to compile yourself (as a Go program with no C linkage).

**Comment by afbjorklund** [comment]
It is the same file in all packages anyway, so I don't think it is important which distribution you choose.

* https://github.com/Mirantis/cri-dockerd/issues/276#issuecomment-1820382923

EDIT: Actually the "jammy" package was recompiled with a later glibc, so it is an exception.

* https://github.com/Mirantis/cri-dockerd/issues/361
